testsuite/MDAnalysisTests/topology/PR3959_test.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
This can become part of the other LAMMPS Topology tests in `test_lammpsdata.py` it doesn't need its own file.